src/Compilers/CSharp/Test/Semantic/Semantics/ReadOnlyStructsTests.cs

```diff
@@ -774,16 +1233,16 @@ public struct S
         [Fact]
         public void RefReturningReadOnlyMethod()
         {
-            // PROTOTYPE: would be good to add some more mutation here
-            // as well as expected diagnostics once that part of the feature is ready.
             var csharp = @"
 public struct S
 {
     private static int f1;
-    public readonly ref int M1() => ref f1;
+    public readonly ref int M1_1() => ref f1; // ok
+    public readonly ref readonly int M1_2() => ref f1; // ok
```

Comment:
>readonly ref readonly [](start = 11, length = 21)

This is C++ `const`-level bad. Really disagreeing with LDM here.

Reply:
And for the record: when we were talking about this feature offline, I got which `readonly` meant what wrong.

---
In reply to: [266668722](https://github.com/dotnet/roslyn/pull/34022#discussion_r266668722) [](ancestors = 266668722)